cmd/run.go

```diff
@@ -34,37 +31,19 @@ func newRunCommand() *cobra.Command {
 			return run(flags, args)
 		},
 	}
-	cmd.Flags().StringVar(&flags.file, "file", flags.file, "the path to the dianostics script file to run")
-	cmd.Flags().StringVar(&flags.output, "output", "", "the path of the generated archive file")
+	cmd.Flags().StringVar(&flags.file, "file", flags.file, "the path to the diagnostics script file to run")
+	//cmd.Flags().StringVar(&flags.output, "output", "", "the path of the generated archive file")
```

Comment:
Like I mentioned, since this flag functionality can be overridden by the `crashd_config` built-in, we should remove this flag entirely. Waiting on your suggestion

Reply:
@srm09 Let's leave it.  The intent is that a script can be configured without having to edit the script directly.  For instance, the following command 

```sh
crashd run --file diagnostics.file --workdir /some/dir --output file.gz
```
Now, the script can be written 
```python
crashd_config(workdir=args.workdir, output=args.output)
```
Also, in the code, the `crashdFunc` built-in Go function could be updated to look for program CLI arg flags to be used as default.